kagami/core/supervisor.py

```diff
@@ -208,3 +225,64 @@ async def list_resource(self) -> dict[str, ResourceInfo]:
     @staticmethod
     def _parse_address(host: str, port: int):
         return f"{host}:{port}"
+
+    async def get_announcement(self):
+        # 这里应该实现从数据库或缓存中获取最新公告的逻辑
+        # 临时返回一个示例公告
+        return Announcement(
+            content="Welcome to Kagami!",
```

Comment:
DO **NOT** hard code example.

I assume you didn't see [previous comment](https://github.com/ProjectSoftoid/Kagami/pull/17#discussion_r1860667383), so I'm saying it again.